### ISSUE TITLE
Arena Alt+F4 exploit

### ISSUE THREAD

**Issue by popalexandrudan** [SunwellWoW/Sunwell-TBC-Bugtracker#3339]
Description: 
You can abuse arena loss by doing alt+f4 and the game won't count.

How it works: 
If you alt+f4 during arena, the game never ends.

How it should work: 
 :)

Source (you should point out proofs of your report, please give us some source):

**Comment by ujellyx** [comment]
/confirm - as the games won't count and you don't lose rating, it's pretty much exploitable to get to a higher rating more quickly (and even with negative win-loss-ratio).

**Comment by Aikamz** [comment]
Yes, there are loads of people abusing this bug to get 2k rating.
